intel-sgx/ppid-decryption-tool/Dockerfile

```diff
@@ -0,0 +1,99 @@
+From ubuntu:focal as sgx_sdk
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND="noninteractive" TZ="America/New_York" apt-get install -y git build-essential python
+
+# Install user
+RUN useradd -rm -d /home/ppid-tool -s /bin/bash -g root -G sudo -u 1000 ppid-tool
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN echo 'ppid-tool:ppid-tool' | chpasswd
+USER ppid-tool
+WORKDIR /home/ppid-tool
+
+# Install SGX Dev tools
```

Comment:
Is it necessary to build & install sgx sdk from source?
Could we just install the dev pkgs like `libsgx-dcap-ql`
Note: I lack knowledge of how build works here.

Reply:
sdk env is needed to build the project

Reply:
Most of the tools installed here, isn't needed. I think [this](https://github.com/intel/linux-sgx/blob/main/linux/installer/docker/Dockerfile#L32-L48) is only what's needed from Intel (but you need to update the distro used).